### PR TITLE
[HPT-1235] fix layout violations (according to rubocop).

### PR DESCRIPTION
**Pull request by mwoodiupui**
Oh, this also fixes bundler complaints (mostly sorting).

### CODE REVIEW

File: lib/iu_metadata/marc_record.rb [hunk 1]
```diff
@@ -209,10 +210,12 @@ def title(include_initial_article = true)
 
     def subject
       # Broken: name puctuation won't come out correctly
-      formatted_fields_as_array([
+      formatted_fields_as_array
+      ([
```
Comment (dlpierce) [inline lib/iu_metadata/marc_record.rb]: I think this being on a new line would cause the method to be called without any arguments.

**Comment by mwoodiupui** [comment]
Thank you @dlpierce.  Of course a duck-typed language would need to parse that way.  Moving the parens fixed it right up.

**Comment by mwoodiupui** [comment]
Rebased to pick up conflicting Gemfile changes.

**Comment by mwoodiupui** [comment]
Rebased again.

**Comment by mwoodiupui** [comment]
Rebased *again*.

**Comment by mwoodiupui** [comment]
Rebased **again**.

**Review by aploshay — APPROVED** [review]
Approved

**Pull request merged**